src/__tests__/utils/amplify-utils.test.ts

```diff
@@ -33,3 +43,23 @@ test('getAmplifyBackendClient', () => {
     authMode: 'iam',
   });
 });
+
+test('getAmplifyBackendClient without proper config params', () => {
+  process.env.NEXT_PUBLIC_FEATURE_VISIBILITY_TESTING = 'on';
```

Comment:
Please can you move this down a line, as it forms part of "arranging" the test.

Reply:
Done